seed/static/seed/partials/dataset_list.html

```diff
@@ -32,8 +32,8 @@ <h1><i class="fa fa-sitemap"></i>
                             </span>
                             <form class="row action edit_file_name form form-inline edit_import_meta_form" role="form" sd-enter="save_dataset_name(d)" ng-show="d.edit_form_showing">
                                 <div class="form-group col-sm-8">
-                                    <label class="sr-only" for="editDatasetName" translate>Edit Name</label>
-                                    <input class="form-control input-sm" id="editDatasetName" type="text" name="name" ng-model="d.name" style="width: 100%;">
+                                    <label class="sr-only" for="editDatasetName-{$ $index $}" translate>Edit Name</label>
+                                    <input class="form-control input-sm" id="editDatasetName-{$ $index $}" type="text" name="name" ng-model="d.name" style="width: 100%;">
```

Comment:
This is nothing to do with my work in this PR, but when I was debugging, I kept getting a bunch of DOM errors every time I had more than one data set, because the id was not unique.  I added this angular index.  The error no longer appears, but I didn't do any additional testing.  @axelstudios is this OK?